src/modals/stop-modal/stop.modal.html

```diff
@@ -35,5 +35,5 @@
       </ion-checkbox>
     </ion-item>
   </ion-list>
-  <pagination-controls center class="pages" (pageChange)="p = $event"></pagination-controls>
```

Comment:
Idk what this `center` is for... didn't seem to center the pagination

Reply:
Oops, someone meant `text-center` (or this syntax has been deprecated, etc.). Now it "works".